spec/tmp/validator.spec.ts

```diff
@@ -0,0 +1,46 @@
+/*Use the fhir validator jar to check the researchstudy /bundle being sent to the UI is formatted properly and satisfy fhir standards
+
+Download the fhir validator here: https://storage.googleapis.com/ig-build/org.hl7.fhir.validator.jar and place in this directory
+
+Paste an example research study in the resource.json file before running the test
+
+*/
+
+import { exec } from "child_process";
+
+//NOTE: The jar file must be named org.hl7.fhir.validator.jar
+
+
+
+
+describe("Fhir Validator jar", () => {
+
+    /*This test case is temporarily disabled. Change xit --> it to run this test case.
+      NOTE: The test must be disabled (changed to xit) on commit 
+    */
+   
+   beforeEach(function() {
+       
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
+    });
+    afterEach(function(){
+        jasmine.DEFAULT_TIMEOUT_INTERVAL=5000;
+    });
+
+    xit("validates sample fhir object" ,   function (done) {
+
```

Comment:
The code could use some formatting cleanup, but this also doesn't appear to actually check to see if there's an error from the validator - the test would pass even if the validator finds an error.